src/main.c

```diff
@@ -961,7 +961,7 @@ static int listener_setup_ssl(h2o_configurator_command_t *cmd, h2o_configurator_
         goto Error;
     }
     if (use_picotls) {
-        if (cipher_suite_tls13 != NULL && (tls13_cipher_suites = parse_tls13_ciphers(cmd, *cipher_suite_tls13)) == NULL) {
+        if (cipher_suite_tls13_node != NULL && (cipher_suite_tls13 = parse_tls13_ciphers(cmd, *cipher_suite_tls13_node)) == NULL) {
             h2o_configurator_errprintf(cmd, *cipher_suite, "failed to parse the TLS 1.3 cipher suite\n");
```

Comment:
@kazuho Do you want to change `*cipher_suite` here to `*cipher_suite_tls13_node`?

Reply:
Nice catch! This is a crashing bug when `cipher_suite` is NULL. Please see #2654.